### PR TITLE
[SwiftASTContext] Hoist LoadOneModule.

### PR DESCRIPTION
**Pull request by dcci**
This way we can call it from other context(s).

### CODE REVIEW

**Comment by dcci** [comment]
@swift-ci please test and merge

File: include/lldb/Symbol/SwiftASTContext.h [hunk 1]
```diff
@@ -769,6 +771,14 @@ class SwiftASTContext : public TypeSystem {
 
   void SetCachedType(const ConstString &mangled, const lldb::TypeSP &type_sp);
 
+  static bool
```
Comment (jimingham) [inline include/lldb/Symbol/SwiftASTContext.h:774]: Is there any reason why these have to be static methods?  They have to have a swift_ast_context to work, and they are in SwiftASTContext.h.  Why not make them instance methods?

Comment (adrian-prantl) [reply include/lldb/Symbol/SwiftASTContext.h:774]: A static method has the advantage of explicitly declaring what it modifies and makes it a bit easier to read the code if you are unfamiliar with it. That said; if one argument is `this` and it's not const, then this argument doesn't hold up :-)

Comment (dcci) [reply include/lldb/Symbol/SwiftASTContext.h:774]: You are correct, I'll constify `SwiftASTContext` here.

Comment (jimingham) [reply include/lldb/Symbol/SwiftASTContext.h:774]: Not sure I understand this argument.  This function is loading a module into the AST context.   Conceptually this is not a const operation even if the compiler can't tell that.

**Pull request merged**